templates/archive-tsml-ui-blocks.php

```diff
@@ -0,0 +1,27 @@
+<!-- USE WITH TSML UI IN A BLOCK THEME  -->
+<!doctype html>
+<html <?php language_attributes(); ?>>
+<head>
+	<meta charset="<?php bloginfo( 'charset' ); ?>">
+	<?php wp_head(); ?>
+</head>
+<body <?php body_class(); ?>>
+<?php wp_body_open(); ?>
+<div class="wp-site-blocks">
+
+<?php
+echo do_blocks( '<!-- wp:template-part {"slug":"header","theme":"twentytwentytwo","tagName":"header","className":"site-header","layout":{"inherit":true}} /-->' );
+//block_header_area();
+
+echo do_shortcode('[tsml_ui]');
```

Comment:
would skip an execution step if it were simply `echo tsml_ui();`

Reply:
This is temporary code I borrowed from a wordpress block tutorial ('Creating block templates for custom post types') by Carolina Nymark. It addresses an identified block header part issue which has not been fixed yet the Gutenberg team. There's an obvious problem in that it will only work with the twentytwentytwo theme. I will give your suggestion a shot and see if we can live with the results.

Reply:
i was suggesting replacing `echo do_shortcode('[tsml_ui]');` with `echo tsml_ui();` because these lines are equivalent -- sorry if the wrong lines got highlighted.